blockly-core/core/ui/block_svg/block_svg.js

```diff
@@ -97,7 +97,7 @@ Blockly.BlockSvg.prototype.init = function() {
 /**
  * Add or remove the UI indicating if this block is movable or not.
  */
-Blockly.BlockSvg.prototype.updateMovable = function() {
+Blockly.BlockSvg.prototype.updateMovable = function () {
```

Comment:
I think Hound is still commenting here because https://github.com/code-dot-org/code-dot-org/blob/half_refactor_please_dont_merge/.hound.yml has the old JavaScript settings.  It will go away once branches referencing the old value expire.